go.mod

```diff
@@ -0,0 +1,5 @@
+module github.com/ccoveille/go-safecast
+
+go 1.23.0
```

Comment:
1.23.1 is out

(and personally I would put just 1.23 so you don't have to keep changing it)

Reply:
or more importantly... you don't want to force users to use 1.23, I wouldn't yet for anything prod

Reply:
That's right, 1.22 is enough

I simply ran `go mod init` that set it up like this

Apparently, I'm still go 1.23.0 locally

Reply:
Fixed in 328cfff6dcfa7086269567cd9c0fba206734019a